packages/client/src/svelte/components/Main/Bars/OrderBar.svelte

```diff
@@ -21,7 +20,7 @@
         NONE
       {:else}
         {$playerOrder?.order.amount / UI_SCALE_FACTOR}
-        {MATERIAL_TYPE[$playerOrder?.order.materialType]}
+        {$materialMetadata[$playerOrder?.order.materialId]?.name}
```

Comment:
not a fan of this being repeated in various places, it could be a function, but not sure how you wanna do this or where to put it